### PR TITLE
Events page updates.

### PR DESCRIPTION
**Pull request by ManuCastrillonM**
- fix height for eventMembers card
- sponsors url now open the correct page
- divide the event sponsors per category

### CODE REVIEW

File: pages/events/_event.vue [hunk 1]
```diff
@@ -53,14 +53,17 @@
...
       <v-container pa-4>
-        <v-layout row wrap justify-space-around>
-          <v-flex xs12 sm3 v-for="sponsor in currentEvent.sponsors" :key="sponsor.name">
-            <div class="vv-day-sponsor px-4 mb-3">
-              <a :href="sponsor.website" target="_blank">
-                <img :src="sponsor.img" class="vv-day-sponsor-logo">
-              </a>
-            </div>
-          </v-flex>
+        <v-layout column v-for="category in sponsors" :key="category.name">
```
Comment (NataliaTepluhina) [inline pages/events/_event.vue]: I don't think we're going to have sponsors categories for events, it seems way too detailed for me.
@jlooper What do you think?

File: components/EventMembers.vue [hunk 2]
```diff
@@ -2,7 +2,7 @@
   <v-container pt-4 pb-5>
     <v-layout wrap justify-space-around>
       <v-flex xs12 sm3 md2 pa-2 v-for="member in members" :key="member.name">
-        <v-card>
+        <v-card class="event-member-card">
```
Comment (NataliaTepluhina) [inline components/EventMembers.vue]:
```suggestion
        <v-card height="100%">
```

File: components/EventMembers.vue [hunk 3]
```diff
@@ -47,4 +47,8 @@ a,
 p {
   width: 100%;
 }
+
+.event-member-card {
```
Comment (NataliaTepluhina) [inline components/EventMembers.vue]: We can delete this, as `v-card` height can be set without a CSS selector

**Comment by NataliaTepluhina** [comment]
@ManuCastrillonM To sum both PRs up and speed up the process I've done some changes directly to `master`

File: pages/events/_event.vue (hunk 1 above)
Comment (jlooper) [reply pages/events/_event.vue]: hi, I believe that having sponsor categories for events will be hard to standardize across all the events thrown by all our chapters. What I recommend is, if you have a sponsor category for your event, put that information in the text of your sponsor description like:

Angel Bread Company
Gold Sponsor for Food 
Angel provides us with delicious brioches for our events. Learn more about them at sponsorwebsite.com

**Comment by ManuCastrillonM** [comment]
Perfect! I'll close the PR.
Thank you ✨

**Pull request closed without merging**